SevenSlots/SevenSlots/SevenSlots/View/CrashView.xaml

```diff
@@ -34,7 +34,7 @@
                     </CollectionView.Background>
                     <CollectionView.ItemsLayout>
                         <LinearItemsLayout   Orientation="Horizontal"/>
```

Comment:
extra space between the tag and the attribute

Reply:
I didn't even changed this code, this was from Tinel...